lib/irb/cmd/show_source.rb

```diff
@@ -27,11 +27,18 @@ def execute(str = nil)
           puts "Error: Expected a string but got #{str.inspect}"
           return
         end
-
-        source = SourceFinder.new(@irb_context).find_source(str)
+        if str.include? " -s"
+          str, esses = str.split(" -")
+          s_count = esses.count("^s").zero? ? esses.size : 1
+          source = SourceFinder.new(@irb_context).find_source([str, s_count])
+        else
+          source = SourceFinder.new(@irb_context).find_source(str)
+        end
 
         if source
           show_source(source)
+        elsif s_count
+          puts "Error: Couldn't locate a definition for #{str}"
```

Comment:
Should it be this instead?
```suggestion
          puts "Error: Couldn't locate a super definition for #{str}"
```

Reply:
Hmm, I'm not sure honestly.  Want me to ask on Mastodon and see what feedback I get?

Reply:
If we don't want to provide a different message for super definition, why do we need this `elsif` branch then 🤔 The message is exactly the same as the `else` condition's.

Reply:
Lol, fair point!

Well what do you think?  Is it more helpful with the `elsif` branch adding `super` to the Error message or simply removing the `elsif` and having it return as it normally would when it hits the `else`?

Reply:
I think mentioning super is necessary because we do find the signatured method's source.

For example, in Pry:

- If it doesn't find the method's source, it prints `Error: Couldn't locate a definition for Foo#baz`
- But if it doesn't find the method's super method, it prints: `Error: No superclass found for #<UnboundMethod: Bar#baz() test.rb:6>`

I think we can tweak the message later, but IMO they have to be different.

Reply:
Makes sense, changing now.